spec/services/importers/create_npq_contract_spec.rb

```diff
@@ -55,6 +56,36 @@
       end
     end
 
+    context "when new version" do
+      let(:set_to_latest_version) { true }
```

Comment:
suggestion: Maybe if we add 
```
let(:contract_version) { "0.0.3" }
let!(:statement) { create(:npq_statement, cpd_lead_provider:, cohort:, contract_version: contract_version) }
```
and remove line https://github.com/DFE-Digital/early-careers-framework/pull/3844/files#diff-dd0f018c7180c0386edef9776579c09f54dff0c176a6b58f2d9475705459bfe9R74
 The test will be more explicitly testing for the newer version as we currently aren't setting up any statements in the spec

Reply:
That would be better, let me make add this change to be more specific to newer versions.